searx/templates/__common__/new_issue.html

```diff
@@ -0,0 +1,74 @@
+{% macro new_issue(new_issue_url, engine_name, engine_reliability) %}
+<form action="{{ new_issue_url }}" method="GET">
+    <input name="title" type="hidden" value="Bug: {{ engine_name }} engine">
+    <input name="labels" type="hidden" value="bug">
+    <input name="template" type="hidden" value="bug-report.md">
+    <textarea name="body" style="display: none;">{{- '' -}}
+
+**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**
+<!-- If you are running on master branch using git execute this command
+in order to fetch the latest commit ID:
+```
+git log -1
+``` 
+If you are using searx-docker then look at the bottom of the Searx page
+and check for the version after "Powered by searx"
+
+Please also stipulate if you are using a forked version of Searx and
+include a link to the fork source code.
```

Comment:
I do not know the state of docker image from searxng, but I would suggest to change this into "Powered by searxng".

I also suggest to modífy (into) ".. using forked version of searxng  ..."